### ISSUE TITLE
doesn't support arabic

### ISSUE THREAD

**Issue by musabkhunaijir** [Trott/node-slug#1]

**Comment by Trott** [comment]
Can you provide some sample code showing the problem?  Is it that provided Arabic characters results in an error or bad results or something else?

**Comment by musabkhunaijir** [comment]
`var slug = require("slug")

console.log(slug("hi there"));
//"hi-there"


console.log(slug("مرحبا بك"));
// ""
`

**Comment by Trott** [comment]
Thanks for reporting the issue. I just published version 1.1.0 which adds support for Arabic.

```js
var slug = require("slug")

console.log(slug("hi there"));
//"hi-there"

console.log(slug("مرحبا بك"));
// "mrhb-bk"
```

**Comment by musabkhunaijir** [comment]
thanks
